asreview/metrics.py

```
@@ -35,7 +36,8 @@ def loss(labels: list[int]):
     Ny = sum(labels)
     Nx = len(labels)
     if Ny == 0 or Nx == Ny:
-        raise ValueError("Labels must contain two distinct classes.")
+        warnings.warn("Labels do not contain two distinct classes. Returning NaN.")
+        return np.nan
```

This should raise an error instead of a warning.